test/unit/Ramm/swap.js

```diff
@@ -404,4 +405,126 @@ describe('swap', function () {
+    const [member] = fixture.accounts.members;
+
+    const nxmIn = parseEther('1');
+    const minAmountOut = parseEther('0.015'); // 0.0152 spotB
+
+    const { timestamp } = await ethers.provider.getBlock('latest');
+    const nextBlockTimestamp = timestamp + 10;
+    const deadline = nextBlockTimestamp + 5 * 60;
+
+    const state = await getStateAtBlockTimestamp(ramm, pool, mcr, tokenController, nextBlockTimestamp);
+    const { ethOut } = getExpectedStateAfterSwapNxmForEth(state, nxmIn);
+
+    const ethReleasedBefore = await ramm.ethReleased();
+
+    await setNextBlockTime(nextBlockTimestamp);
+    await ramm.connect(member).swap(nxmIn, minAmountOut, deadline, { maxPriorityFeePerGas: 0 });
+
+    const ethReleasedAfter = await ramm.ethReleased();
+    const expectedEthReleasedAfter = ethReleasedBefore.add(ethOut);
+
+    expect(ethReleasedAfter).to.be.equal(expectedEthReleasedAfter);
+  });
+
+  it('should increase nxm circuit breaker accumulator nxmReleased', async function () {
+    const fixture = await loadFixture(setup);
+    const { ramm, pool, mcr, tokenController } = fixture.contracts;
+    const [member] = fixture.accounts.members;
+
+    const ethIn = parseEther('1');
+    const minAmountOut = parseEther('0');
+
+    const { timestamp } = await ethers.provider.getBlock('latest');
+    const nextBlockTimestamp = timestamp + 10;
+    const deadline = nextBlockTimestamp + 5 * 60;
+
+    const state = await getStateAtBlockTimestamp(ramm, pool, mcr, tokenController, nextBlockTimestamp);
+    const { nxmOut } = getExpectedStateAfterSwapEthForNxm(state, ethIn);
+
+    const nxmReleasedBefore = await ramm.nxmReleased();
+
+    await setNextBlockTime(nextBlockTimestamp);
+    await ramm.connect(member).swap(0, minAmountOut, deadline, { value: ethIn, maxPriorityFeePerGas: 0 });
+
+    const nxmReleasedAfter = await ramm.nxmReleased();
+    const expectedNxmReleasedAfter = nxmReleasedBefore.add(nxmOut);
+
+    expect(nxmReleasedAfter).to.be.equal(expectedNxmReleasedAfter);
+  });
+
+  it('should emit EthSwappedForNxm when successfully swapped ETH for NXM', async function () {
+    const fixture = await loadFixture(setup);
+    const { ramm, pool, mcr, tokenController } = fixture.contracts;
+    const [member] = fixture.accounts.members;
+
+    const ethIn = parseEther('1');
+    const minAmountOut = parseEther('28.8');
+
+    const { timestamp } = await ethers.provider.getBlock('latest');
+    const nextBlockTimestamp = timestamp + 5 * 60;
+    const deadline = nextBlockTimestamp + 5 * 60;
+
+    const state = await getStateAtBlockTimestamp(ramm, pool, mcr, tokenController, nextBlockTimestamp);
+    const { nxmOut } = getExpectedStateAfterSwapEthForNxm(state, ethIn);
+
+    await setNextBlockTime(nextBlockTimestamp);
+    const swap = ramm.connect(member).swap(0, minAmountOut, deadline, { value: ethIn, maxPriorityFeePerGas: 0 });
+    await expect(swap).to.emit(ramm, 'EthSwappedForNxm').withArgs(member.address, ethIn, nxmOut);
+  });
+
```

Comment:
seems this is a [duplicated test case](https://github.com/NexusMutual/smart-contracts/blob/2e683ac3fa42ef5f16e1fef3b6d6185a35cb7bf0/test/unit/Ramm/swap.js#L317)

Reply:
Removed